tools/stimulus/functions-R.yaml

```diff
@@ -55,7 +55,7 @@ igraph_get_all_eids_between:
     DEPS: from ON graph, to ON graph, eids ON graph
 
 igraph_incident:
-    IGNORE: RR, RC
+    DEPS: vid ON graph
```

Comment:
@szhorvat is this a DEPS that should be committed to the C library instead of here?

Reply:
https://github.com/igraph/igraph/pull/2857